#!/usr/bin/env python
# field of view configuration

PACKAGE="gravity"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("gravity_damp", int_t, 1, "gravity_damp", 0, 5, 100)
gen.add("lean_tolerance", double_t, 1, "lean_tolerance", 5.0, 0.0, 90.0)

exit(gen.generate(PACKAGE, "gravity", "gravity_node"))
